osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/scheduling/OsgpScheduler.java

```diff
@@ -131,7 +142,8 @@ public Trigger createJobTrigger(
    * @return A {@link Trigger} instance.
    */
   public Trigger createJobTrigger(final JobDetail jobDetail, final String cronExpression) {
```

Comment:
Is this method made public only to be available for test?

Reply:
No we use it in SMHE